test/images/pets/peer-finder/BASEIMAGE

```diff
@@ -0,0 +1,4 @@
+amd64=gcr.io/google-containers/debian-base-amd64:0.1
```

Comment:
these should be updated to 0.2 now.

Reply:
yup @mkumatag Please fix

Reply:
@ixdy @luxas done the changes as suggested PTAL.